subprojects/gradle/gradle-logger/src/main/kotlin/com/avito/logger/LoggerRegistry.kt

```diff
@@ -0,0 +1,100 @@
+package com.avito.logger
+
+import com.avito.android.elastic.ElasticConfig
+import com.avito.android.sentry.SentryConfig
+import com.avito.logger.destination.ElasticDestination
+import com.avito.logger.destination.SentryDestination
+import com.avito.logger.destination.Slf4jDestination
+import com.avito.logger.formatter.AppendMetadataFormatter
+import com.avito.logger.handler.CombinedHandler
+import com.avito.logger.handler.DefaultLoggingHandler
+import com.avito.logger.handler.LoggingHandler
+
+object LoggerRegistry {
```

Comment:
Why not `LoggerFactory`? What is the difference?

Reply:
fixed https://github.com/avito-tech/avito-android/pull/695/commits/f3c7f00de17988ef0fdf192c0ccc799126535057